app/src/androidTest/java/com/example/triptracker/e2eTest.kt

```diff
@@ -121,19 +261,5 @@ class E2ETest {
     // Go back and assert we are in the profile screen
     composeTestRule.onNodeWithTag("GoBackButton").performClick()
     composeTestRule.onNodeWithTag("ProfileOverview").assertIsDisplayed()
-
```

Comment:
Why are these lines removed?

Reply:
This was an accident on my side, let's fix that !